includes/export/abstract-wc-csv-exporter.php

```diff
@@ -330,6 +330,8 @@ public function escape_data( $data ) {
 			$data = "'" . $data;
 		}
 
+		$data = str_replace( '"', '""', $data );
```

Comment:
Ill send you the data causing the issue then - I pulled that code from the CSV import suite.